apps/asap-server/test/handlers/calendar/subscribe-handler.test.ts

```diff
@@ -87,15 +87,13 @@ describe('Calendar handler', () => {
       );
     });
 
-    test('Should return "ERROR" and alert when the subscription was unsuccessful, but should not throw', async () => {
+    test('Should return 502 and alert when the subscription was unsuccessful', async () => {
```

Comment:
to be fair that's not really what it does but I guess you wanted these tests to follow the same pattern - the only I see in the expectation is that it throws an error